pkg/sql/run_control_test.go

```diff
@@ -326,7 +330,11 @@ GRANT admin TO has_admin2;
 					// end of the test.
 					errRE = "sql: database is closed"
```

Comment:
i saw that typo ;)

Reply:
ur a wizard!

Reply:
i did that to make sure the assertion was actually being called haha